src/java.base/share/classes/sun/util/calendar/CalendarUtils.java

```diff
@@ -25,7 +25,9 @@
 
 package sun.util.calendar;
 
-public class CalendarUtils {
+public final class CalendarUtils {
+
+    private CalendarUtils() {}
```

Comment:
It may be obvious, but adding a comment would be helpful, i.e., no instance may be created.